app/assets/stylesheets/components/pills.scss

```diff
@@ -0,0 +1,58 @@
+////////////////////////////////////////////////////////////
+// .c-pill /////////////////////////////////////////////////
```

Comment:
This will hopefully be replaced with https://github.com/forem/forem/pull/16271 when merged...